libsolidity/interface/OptimiserSettings.h

```diff
@@ -40,7 +40,7 @@ struct OptimiserSettings
 
 			// should have good "compilability" property here.
 
-			"eul"                      // Run functional expression inliner
+			"Feul"                     // Run functional expression inliner
```

Comment:
Shouldn't we rather run it before the full inliner instead of the expresison inliner?

Reply:
Ah, you are saying that if the function only returns a single variable, then the expression inliner can also do it. But maybe better to run it before both the functional inliner and the full inliner?